styles/globals.css

```diff
@@ -95,7 +95,7 @@ header {
 }
 
 #nav .logo {
-  background: url('../public/newlogo.png') no-repeat center;
+  background: url('../public/betterbean-logo.webp') no-repeat center;
```

Comment:
It's great that we're using webp, and I would also like us to have a fallback option if the users' browser does not support webp.
https://css-tricks.com/using-webp-images/